### ISSUE TITLE
artifact-poisoning-medium: 3-entry UntrustedTriggers misses issues/discussion_comment/pull_request_review workflows (false negative)

### ISSUE THREAD

**Issue by on-keyday** [ultra-supara/sisakulint#534]
## Summary

`artifact-poisoning-medium`'s trigger gate `UntrustedTriggers` (`pkg/core/artifactpoisoningutil.go`) contains only 3 entries: `workflow_run`, `pull_request_target`, `issue_comment`. The shared `PrivilegedTriggers` set has 6 (`issues`, `discussion_comment`, `pull_request_review` in addition). As a result, the same poisonable artifact-download step is flagged under `workflow_run` but produces **zero findings under `issues`** (and the other omitted triggers) — a false negative, not a severity downgrade. Affected version: `main` @ `dfeb590` (re-verified 2026-07-07).

## Reproduce

Two workflows identical except for the trigger:

```yaml
# on: issues (types: [opened])         → 0 artifact-poisoning-medium findings
# on: workflow_run (types: [completed]) → 1 artifact-poisoning-medium finding
jobs:
  dl:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v4
      - uses: dawidd6/action-download-artifact@v2
        with:
          name: pr_number
      - run: sh cmd.sh
```

## Suggested direction

Align `UntrustedTriggers` with the shared trigger sets, or document why the 3-entry set is intentional. At minimum the omission of `issues` / `discussion_comment` looks unintentional (both are in every other privileged/untrusted set). Whether `pull_request_review` belongs here is a separate design choice — cf. the #505 resolution, which deliberately routes that trigger to medium severity for the injection rules rather than dropping it.

Related to #502 pattern 2 (cross-rule consistency drift; same shape as #505, different rule family).
